service/src/main/kotlin/io/provenance/digitalcurrency/consortium/bankclient/BankClient.kt

```diff
@@ -22,6 +22,9 @@ interface BankClient {
     @RequestLine("POST /api/v1/mints/complete/{uuid}")
     fun completeMint(@Param("uuid") uuid: UUID, request: EmptyRequest = EmptyRequest()): ResponseEntity<String>
 
+    @RequestLine("POST /api/v1/burns/complete/{uuid}")
```

Comment:
will need to update cme ops to accept this for local testing

Reply:
adding, will push today or tomorrow